index.html

```diff
@@ -1,214 +1,212 @@
-
-<p><strong>TabFern is now available in the <a href="https://chrome.google.com/webstore/detail/tabfern-tab-manager-and-b/hbajjpcdbninabigakflkhiogmmjaakm">Chrome Web Store</a></strong></p>
-
-<p>Check out the code or report issues <a href="https://github.com/cxw42/TabFern">on GitHub</a>.</p>
-
-<h1 id="usage">Usage</h1>
-
-<ul>
-  <li>Click the fern icon (<img src="/TabFern/assets/fern16icon.png" alt="image" />)
-in the Chrome toolbar to open the TabFern view.
-The view will also open when you start Chrome.
-    <ul>
-      <li>If you can’t find the TabFern window, double-click the fern icon in the
-taskbar in any Chrome window.  That will summon the TabFern window
-to that Chrome window.</li>
-    </ul>
-  </li>
-  <li>
-    <p>The TabFern view shows a list of all your open windows and tabs, in white.
-It shows you saved windows and tabs in gray.</p>
-  </li>
-  <li>
-    <p>You can control other Chrome windows from the TabFern window by
-right-clicking their entries, or by clicking the corresponding
-buttons that appear at the right when you hover
-the mouse pointer over the window’s name.</p>
-  </li>
-  <li>
-    <p>To mark a window to be saved, you have two choices:</p>
-
-    <ol>
-      <li>Give the window a name using the pencil icon
-(<img src="/TabFern/assets/icons/pencil.png" alt="image" />).</li>
-      <li>Hit the middle icon showing a rectangle with a red dot
-(<img src="/TabFern/assets/icons/picture_delete.png" alt="image" />).  The window will close.</li>
-    </ol>
-  </li>
-  <li>
-    <p>Folder icons are:</p>
-
-    <ul>
-      <li>Open, unsaved: a monitor
-(<img src="/TabFern/assets/icons/monitor.png" alt="image" />)</li>
-      <li>Open, saved: a monitor with a green dot
-(<img src="/TabFern/assets/icons/monitor_add.png" alt="image" />).</li>
-      <li>Closed, saved (closed unsaved aren’t in the tree): a white file folder.</li>
-    </ul>
-  </li>
-  <li>
-    <p>Saved windows will be saved even if you close them manually.  To remove them
-from the tree, hit the delete icon (red X,
-<img src="/TabFern/assets/icons/cross.png" alt="image" />).</p>
-
-    <ul>
-      <li>If you accidentally delete a window, look on the main menu (bottom-right
-of the TabFern window — see next section).
-There will be a “Restore last deleted” option that will
-bring the tabs back.</li>
-    </ul>
-  </li>
-  <li>
-    <p>To change the text size in the TabFern window, click into the window,
-then hit Control+Plus for larger or Control+Minus for smaller.  This is the
-normal Google Chrome browser zoom.</p>
-  </li>
-  <li>
-    <p>For help, sorting, options, or more functions, click the menu icon
-(<strong>≡</strong>) at the bottom right of the TabFern window.</p>
-  </li>
-  <li>Note: windows you do not expressly save will not be saved when you exit!
-I am open to discussion of better ways to handle this.</li>
-</ul>
-
-<h1 id="backup-and-restore">Backup and Restore</h1>
+<!DOCTYPE html>
+<html style="font-family: 'Josefin Sans', sans-serif;">
```

Comment:
I noticed that there are a lot of inline `style="..."` attributes in the index.html.  Would you please move them to the CSS file, and add `class=` or `id=` tags here for the CSS to reference?  That way the styles can be shared between index.html and privacy.html.  This will also make it easier to add other pages in the future.  For example, I may at some point move this theme to [Statocles](https://metacpan.org/pod/Statocles) for easier content generation.

Reply:
The HTML is much easier to read now :+1: